templates/reservation/reservation.py

```diff
@@ -28,11 +27,23 @@ def __init__(self, name, guid=None, data=None):
         self.recurring_action(self._cleanup, 3600*12)  # 12h
 
     def validate(self):
-        if not self.data.get('creationTimestamp'):
-            self.data['creationTimestamp'] = time.time()
+        for key in ['creationTimestamp', 'expiryTimestamp']:
```

Comment:
What about already existing services?
they won't have a value on the expiryTimestamp

Reply:
May be we can have some sort of migration script/function? and for existing active reservations that have a creationtimestamp and no expiry, it sets the expiry to 1 month after the creation? Because I believe the validation needs to be there.

Reply:
> for existing active reservations that have a creationtimestamp and no expiry, it sets the expiry to 1 month after the creation? 

Good idea. So the template can update the data of the services automatically